Gems/Atom/Feature/Common/Code/Source/Material/Transform2DFunctor.cpp

```diff
@@ -39,59 +39,21 @@ namespace AZ
                     ;
             }
         }
-
+        
```

Comment:
Nitpick: Looks like an extra tab space here

Reply:
ugh I think you're right. Not running AR again for that though, I'll fix it in my next PR